### PR TITLE
Added Front End

### PR DESCRIPTION
**Pull request by Adnanhoq**
This PR encompasses the following tickets:
[- Set up Repo](https://kainossoftware.atlassian.net/browse/GAIBP-47)
[- File Upload](https://kainossoftware.atlassian.net/browse/GAIBP-46)

### CODE REVIEW

**Review by Jack-M1 — APPROVED** [review]

**Comment by Adnanhoq** [comment]
Screenshot 2024-10-21 at 13 53 32
Screenshot 2024-10-21 at 15 33 37
Screenshot 2024-10-21 at 15 33 54
Screenshot 2024-10-21 at 15 33 58
Screenshot 2024-10-21 at 15 34 04

File: CrimeLineFrontEnd/src/assets/favicon_io/favicon.ico
Comment (Leebee1392) [inline CrimeLineFrontEnd/src/assets/favicon_io/favicon.ico:1]: Remove

**Review by Leebee1392 — CHANGES_REQUESTED** [review]

File: CrimeLineFrontEnd/src/assets/favicon_io/favicon.ico
Comment (Adnanhoq) [reply CrimeLineFrontEnd/src/assets/favicon_io/favicon.ico:1]: Removed

**Review by Leebee1392 — APPROVED** [review]

File: .gitignore [hunk 1]
```diff
@@ -1 +1,26 @@
-node_modules
\ No newline at end of file
+# Logs
```
Comment (Leebee1392) [inline .gitignore:1]: Why are there two .gitignores

File: CrimeLineFrontEnd/eslint.config.mjs [hunk 2]
```diff
@@ -1,12 +1,12 @@
-import globals from "globals";
-import pluginJs from "@eslint/js";
-import tseslint from "typescript-eslint";
-import pluginReact from "eslint-plugin-react";
+// import globals from "globals";
```
Comment (Leebee1392) [inline CrimeLineFrontEnd/eslint.config.mjs:1]: If they are all commented out, shouldn't the file just be deleted

File: CrimeLineFrontEnd/src/vite-env.d.ts [hunk 3]
```diff
@@ -0,0 +1 @@
+/// <reference types="vite/client" />
```
Comment (Leebee1392) [inline CrimeLineFrontEnd/src/vite-env.d.ts:1]: Is this needed?

**Review by Leebee1392 — CHANGES_REQUESTED** [review]

**Review by Leebee1392 — APPROVED** [review]

**Pull request merged**